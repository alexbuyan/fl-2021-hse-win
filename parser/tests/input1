3 \
1 - 0 \
2 - 2 \
3 - 3 \
1 -> 3 : 1 \
1 -> 2 : 0, 2...9 \
2 -> 2 : 0...9 \
3 -> 2 : 0...9 \