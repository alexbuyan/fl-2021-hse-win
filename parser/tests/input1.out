Alphabet:
	{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}
States:
	(name: 1, type: start)
	(name: 2, type: stock)
	(name: 3, type: terminal)
Edges:
	(from: 1, to: 3, moves: 1)
	(from: 1, to: 2, moves: 0,2,3,4,5,6,7,8,9)
	(from: 2, to: 2, moves: 0,1,2,3,4,5,6,7,8,9)
	(from: 3, to: 2, moves: 0,1,2,3,4,5,6,7,8,9)
